src/util/chunk_fs.js

```diff
@@ -57,6 +61,7 @@ class ChunkFS extends stream.Transform {
                 if (this.q_buffers.length === this.iov_max || this.q_size >= config.NSFS_BUF_SIZE) await this._flush_buffers();
                 chunk = (available_size < chunk.length) ? chunk.slice(available_size) : null;
             }
+            this.total_bytes += this.q_size;
```

Comment:
I would prefer to count it inside flush right next to where we handle _total_num_buffers and offset.
I understand that we are counting the input bytes and the output bytes are the same, only that the timing and error cases is different, so I would count bytes that we already output, and that's only known inside flush.